lib/open_referral_transformer.rb

```diff
@@ -94,6 +97,23 @@ def transform_locations
           collect_phone_data(phone_key: k, phone_hash: v, input: input)
         elsif v["model"] == "postal_address"
           collect_address_data(address_key: k, address_hash: v, input: input)
+        elsif v["model"] == "regular_schedule"
+          if input["Hours of operation"] != nil
+            regex_list = input["Hours of operation"].scan(/\S*day: \S*/)
+            for regex in regex_list do
+              day = regex.split(': ')[0]
+              hours = regex.split(': ')[1]
+              if hours == "Closed"
+                opens_at = nil
+                closes_at = nil
+              else
+                opens_at = hours.split('-')[0]
+                closes_at = hours.split('-')[1]
+              end
+              collect_schedule_data(schedule_key: k, schedule_hash: v, input: input,
+                  day: day, opens_at: opens_at, closes_at: closes_at)
+            end
+          end
```

Comment:
Let's go ahead and pull lines 101-116 in its own method to make this `transform` method more readable. You could call it `process_regular_schedule_text`